data/timeliner.yaml

```diff
@@ -556,6 +556,12 @@ attribute_mappings:
   description: 'Content Deletion Time'
 place_holder_event: true
 ---
+data_type: 'windows:onedrive:log'
+attribute_mappings:
+- name: 'last_written_time'
```

Comment:
last_written_time does not correspond with recorded_time defined in OneDriveLogEvent ?

Reply:
Fixed.